src/EnergyPlus/HWBaseboardRadiator.cc

```diff
@@ -605,7 +605,7 @@ namespace HWBaseboardRadiator {
                 HWBaseboardDesignDataObject.FracRadiant = MaxFraction;
                 thisHWBaseboard.FracConvect = 0.0;
             } else {
-                thisHWBaseboard.FracConvect = 1.0 - AllFracsSummed;
+                thisHWBaseboard.FracConvect = 1.0 - HWBaseboardDesignDataObject.FracRadiant;
```

Comment:
You no longer need line 598 since AllFracsSummed is also set at line 636. I see this same code in ChilledCeilingPanelSimple so what is the plan for other objects?

Reply:
Correction: the code in ChilledCeilingPanelSimple does use FracRadiant in this part of the code.